content/200-concepts/100-components/02-prisma-client/065-read-replicas.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: 'Read replicas'
+metaTitle: 'Read replicas'
+metaDescription: 'This page explains how to set up and use read replicas with Prisma Client'
```

Comment:
I think we should change this to be action focused. "Learn how to set up" rather than "this page explains how to set up"
```suggestion
metaDescription: 'Learn how to set up and use read replicas with Prisma Client'
```